mavros/src/plugins/setpoint_raw.cpp

```diff
@@ -78,8 +78,14 @@ class SetpointRawPlugin : public plugin::PluginBase,
 		auto position = ftf::transform_frame_ned_enu(Eigen::Vector3d(tgt.x, tgt.y, tgt.z));
 		auto velocity = ftf::transform_frame_ned_enu(Eigen::Vector3d(tgt.vx, tgt.vy, tgt.vz));
 		auto af = ftf::transform_frame_ned_enu(Eigen::Vector3d(tgt.afx, tgt.afy, tgt.afz));
-		float yaw = ftf::transform_frame_yaw_ned_enu(tgt.yaw);
-		float yaw_rate = ftf::transform_frame_yaw_ned_enu(tgt.yaw_rate);
+		float yaw = ftf::quaternion_get_yaw(
+					ftf::transform_orientation_aircraft_baselink(
+						ftf::transform_orientation_ned_enu(
+							ftf::quaternion_from_rpy(0.0, 0.0, tgt.yaw))));
+		Eigen::Vector3d ang_vel_enu(0.0, 0.0, tgt.yaw_rate);
+		auto ang_vel_ned = ftf::transform_frame_ned_enu(
+					ftf::transform_frame_aircraft_baselink(ang_vel_enu));
+		float yaw_rate = ang_vel_ned.z();
```

Comment:
`yaw_rate = - tgt.yaw_rate;`

Reply:
Remove `fft::transform_orientation_aircraft_baselink()` part and try again.